universal-application-tool-0.0.1/app/controllers/dev/CloudStorageDevControllerStrategy.java

```diff
@@ -0,0 +1,18 @@
+package controllers.dev;
```

Comment:
just a comment for this file/location-- a little confused why this is being stored in the dev folder since it does seem to be used in non- dev instances?

Reply:
AFAIK this file is only used in dev/FileUploadController. What non-dev instances are you finding concerning? This class is supposed to be dev-mode specific.

Reply:
The name is a little misleading since it's not actually using cloud storage, it's using local storage that simulates cloud storage. Also that it's used in a controller doesn't need to be part of the name. Maybe rename to `CloudEmulatorFileStorageStrategy.java`?

Reply:
Done